js/ui/color_box/color_box.js

```diff
@@ -302,7 +302,8 @@ const ColorBox = DropDownEditor.inherit({
 
     _renderValue: function() {
         const value = this.option('value');
-        this.option('text', this.option('editAlphaChannel') ? colorUtils.makeRgba(value) : value);
+
+        this.option('text', (this.option('editAlphaChannel') && value !== null) ? colorUtils.makeRgba(value) : value);
```

Comment:
Could you simplify this expression? E.g. 
```suggestion
        const convertToColor = value !== null && this.option('editAlphaChannel');
        const text = convertToColor ? colorUtils.makeRgba(value) : value;
        this.option('text', text);
```